addons/l10n_it_edi/models/account_invoice.py

```diff
@@ -171,6 +170,9 @@ def invoice_generate_xml(self):
             invoice.message_post(
                 body=(_("E-Invoice is generated on %s by %s") % (fields.Datetime.now(), self.env.user.display_name))
             )
+    def _render_template(self, template_values):
+        content = self.env.ref('l10n_it_edi.account_invoice_it_FatturaPA_export').render(template_values)
+        return content
```

Comment:
We should make a _get_export_values method and inherit that which recuperates the template_values instead

Reply:
it done.